docs/modules/ROOT/pages/how-tos/metadata/index.adoc

```diff
@@ -1 +1,56 @@
-= Inspecting provenance and attestations
\ No newline at end of file
+= Inspecting provenance and attestations
+
+We are committed to providing exceptional security with {ProductName} by hardening the build platform and providing transparency into the build process and artifact's composition. By hardening the build platform, we enable multiple builds to run on a shared platform while preventing tampering from other running builds (link:https://slsa.dev/spec/v1.1/threats-overview[threats to the build process]). By providing transparency, we enable xref:/advanced-how-tos/managing-compliance-with-ec.adoc[policy evaluation] and auditability on the actual build process (link:https://slsa.dev/spec/v1.1/threats-overview[threats to the package selection]). This commitment means that {ProductName} conforms to the link:https://slsa.dev/spec/v1.0/levels#build-l3[SLSA Build Level 3] requirements.
```

Comment:
The opening here is hard to follow. Mainly, the sentences are too long. Consider this alternative: 
"We are committed to providing exceptional security with {ProductName}. We harden the build platform, we provide transparency into the build process, and we expose composition of artifacts. 

To harden the build platform, we enable multiple builds to run on a shared platform while preventing those builds from tampering with each other. This defends against threats to the build process. To provide transparency, we enable you to evaluate policies and conduct audits on the build process. This defends against threats to package selection. 

Our commitment to security means that {ProductName} conforms to the SLSA Build Level 3 requirements.

Don't take our word for it --you can download and inspect these artifacts yourself!"

Reply:
Yeah, I was having difficulty with this paragraph. I was trying to do some parallel construction but that made it even harder. Are you recommending to remove the links as well? Do they add unnecessary complexity?

Reply:
Ahh nah I think the links are good! I was just being lazy and didn't include them in my alternative.

Also fwiw I think it's particularly difficult to write the opening for this doc. Yours gave a solid starting place/better technical info than what I had there before. I think this draft ^^ just breaks it down makes it more digestible.

Reply:
Something like this?
```suggestion
We are committed to providing exceptional security with {ProductName}. We harden the build platform, we provide transparency into the build process, and we expose the composition of artifacts.

To harden the build platform, we enable multiple builds to run on a shared platform while preventing those builds from tampering with each other. This defends against link:https://slsa.dev/spec/v1.1/threats-overview[threats to the build process]. To provide transparency, we enable you to xref:/advanced-how-tos/managing-compliance-with-ec.adoc[evaluate policies] and conduct audits on the build process. This defends against link:https://slsa.dev/spec/v1.1/threats-overview[threats to the package selection].

Our commitment to security means that {ProductName} conforms to the link:https://slsa.dev/spec/v1.0/levels#build-l3[SLSA v1.0 Build Level 3] requirements.

Don't take our word for it --you can download and inspect these artifacts yourself!
```